From e2c237f489fc2932b75a8b4a0995e5c93c874084 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sat, 13 Jan 2024 21:00:27 +0100
Subject: media: i2c: gc2145: Move upstream driver out of the way

Upstream only implements MIPI CSI, but we already have a driver
with parallel bus support.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/media/i2c/Kconfig                     | 4 ++--
 drivers/media/i2c/Makefile                    | 2 +-
 drivers/media/i2c/{gc2145.c => gc2145-mipi.c} | 0
 3 files changed, 3 insertions(+), 3 deletions(-)
 rename drivers/media/i2c/{gc2145.c => gc2145-mipi.c} (100%)

diff --git a/drivers/media/i2c/Kconfig b/drivers/media/i2c/Kconfig
index e7da18f3da3c..43be07cc5b37 100644
--- a/drivers/media/i2c/Kconfig
+++ b/drivers/media/i2c/Kconfig
@@ -90,9 +90,9 @@ config VIDEO_GC08A3
 	  To compile this driver as a module, choose M here: the
 	  module will be called gc08a3.
 
-config VIDEO_GC2145
+config VIDEO_GC2145_MIPI
 	select V4L2_CCI_I2C
-	tristate "GalaxyCore GC2145 sensor support"
+	tristate "GalaxyCore GC2145 sensor support - MIPI variant"
 	help
 	  This is a V4L2 sensor-level driver for GalaxyCore GC2145
 	  2 Mpixel camera.
diff --git a/drivers/media/i2c/Makefile b/drivers/media/i2c/Makefile
index b429d9f2e982..08bd346db54e 100644
--- a/drivers/media/i2c/Makefile
+++ b/drivers/media/i2c/Makefile
@@ -40,7 +40,7 @@ obj-$(CONFIG_VIDEO_ET8EK8) += et8ek8/
 obj-$(CONFIG_VIDEO_GC0308) += gc0308.o
 obj-$(CONFIG_VIDEO_GC05A2) += gc05a2.o
 obj-$(CONFIG_VIDEO_GC08A3) += gc08a3.o
-obj-$(CONFIG_VIDEO_GC2145) += gc2145.o
+obj-$(CONFIG_VIDEO_GC2145_MIPI) += gc2145-mipi.o
 obj-$(CONFIG_VIDEO_HI556) += hi556.o
 obj-$(CONFIG_VIDEO_HI846) += hi846.o
 obj-$(CONFIG_VIDEO_HI847) += hi847.o
diff --git a/drivers/media/i2c/gc2145.c b/drivers/media/i2c/gc2145-mipi.c
similarity index 100%
rename from drivers/media/i2c/gc2145.c
rename to drivers/media/i2c/gc2145-mipi.c
-- 
2.35.3

